app/src/editor/canvas/components/Embed.jsx

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import React from 'react'
+import { Container } from 'reactstrap'
+import { Link } from 'react-router-dom'
+import { Translate, I18n } from 'react-redux-i18n'
+
+import BodyClass, { PAGE_SECONDARY } from '$shared/components/BodyClass'
+import EmptyState from '$shared/components/EmptyState'
+import Layout from '$mp/components/Layout'
+import links from '../../../links'
+import pageNotFoundPic from '$shared/assets/images/404_blocks.png'
+import pageNotFoundPic2x from '$shared/assets/images/404_blocks@2x.png'
+import styles from './Embed.pcss'
+
+const CanvasEmbedPage = () => (
+    <Layout className={styles.canvasEmbedPage}>
+        <BodyClass className={PAGE_SECONDARY} />
+        <Container>
+            <EmptyState
+                image={(
+                    <img
+                        src={pageNotFoundPic}
+                        srcSet={`${pageNotFoundPic2x} 2x`}
+                        alt={I18n.t('error.notFound')}
+                    />
+                )}
+                link={(
+                    <React.Fragment>
+                        <Link
+                            to={links.userpages.main}
+                            className="btn btn-special"
+                        >
+                            <Translate value="notFoundPage.coreaApp" />
+                        </Link>
+                        <Link
+                            to={links.root}
+                            className="btn btn-special"
+                        >
+                            <Translate value="notFoundPage.publicSite" />
+                        </Link>
+                    </React.Fragment>
+                )}
+                linkOnMobile
+            >
+                <p>Whoops! Embeds are disabled in the beta for now.</p>
```

Comment:
Maybe translate this as well, for consistency?

Reply:
Thought about this and decided to go untranslated since this is a temporary solution after all and won't (hopefully) be there for long.